content/en/about/feature-stages/index.md

```diff
@@ -63,7 +63,7 @@ Below is our list of existing features and their current phases. This informatio
 | [Stackdriver Integration](/docs/reference/config/policy-and-telemetry/adapters/stackdriver/) | Beta
 | [Distributed Tracing to Zipkin / Jaeger](/docs/tasks/observability/distributed-tracing/) | Beta
 | [Logging with Fluentd](/docs/tasks/observability/mixer/logs/fluentd/) | Beta
-| [Trace Sampling](/docs/tasks/observability/distributed-tracing/overview/#trace-sampling) | Beta
+| [Tracing Configurability](/docs/tasks/observability/distributed-tracing/configurability) | Experimental
```

Comment:
I don't think we can do this.

You need to keep Sampling at beta and add another row for Configurability at Experimental.

We can't take away a Beta feature and replace it with Experimental. The "feature" Sampling is still beta.